src/main/kotlin/Network.kt

```diff
@@ -0,0 +1,29 @@
+class Network() {
```

Comment:
Why don't we add the "CarsThatAreAllowedToDrive" to the CSV Input in the main function ? I think what we actually need is a value or something, which describes the delay, what's also missing in our file.